package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "refract-query",
+  "version": "0.0.1",
+  "description": "",
+  "main": "./lib",
+  "scripts": {
+    "prepublish": "not-in-install && npm run build || :",
+    "lint": "eslint src test",
+    "build": "babel src/ --out-dir lib/",
+    "test": "npm run lint && mocha --compilers js:babel-core/register -R spec --recursive"
+  },
+  "engines": {
+    "node": "5.1.x",
+    "npm": "3.3.x"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com:apiaryio/refract-query.git"
+  },
+  "author": "Apiary Inc. <support@apiary.io>",
+  "license": "MIT",
+  "devDependencies": {
+    "babel": "",
+    "babel-cli": "",
+    "babel-core": "",
+    "babel-eslint": "",
+    "babel-preset-es2015": "^6.1.18",
+    "babel-preset-stage-0": "^6.1.18",
+    "babelify": "",
+    "chai": "",
+    "eslint": "",
+    "eslint-config-airbnb": "",
+    "eslint-plugin-react": "^3.10.0",
+    "in-publish": "",
+    "mocha": ""
+  },
+  "babel": {
+    "presets": [
+      "es2015",
+      "stage-0"
+    ]
+  },
+  "dependencies": {
+    "lodash": "^3.10.1"
```

Comment:
Please do not take this as a dependency, but require a lodash instance.See [here](https://github.com/apiaryio/lodash-api-description/blob/master/lib/index.js#L4)

Furthermore (I am sorry to reiterate on this again), I really do not understand why this should be a separate repo and not included with the other mixins.